CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Changes:
+
+    All changes should have an issue in Github that well describes why and what 
+    the change aims to achieve. All changes should be made infeature branches 
+    and shall be code reviewed by at least 2 different companies in the group
+
+# Release:
+    To make a new release, a pull request should be approved by at least 2 
+    different companies in the group. 
+    The release should be taged in git with a release number. 
+    This release number correlate with the version publicized in the IG.
```

Comment:
"Correlate" should be "correlates"

"version publicized" should be "version published"

Reply:
Thanks @vjernelov I have now pushed a change